pkg/deploy/api/types.go

```diff
@@ -437,6 +437,8 @@ type DeploymentCondition struct {
 	Type DeploymentConditionType
 	// Status of the condition, one of True, False, Unknown.
 	Status kapi.ConditionStatus
+	// The last time this condition was updated.
+	LastUpdateTime unversioned.Time
```

Comment:
The former is updated only when the Status of the condition transitions, the latter is updated every time the condition is updated.